packages/components/src/components/Address/Address.stories.tsx

```diff
@@ -26,6 +33,12 @@ const AddressUsingProvider = () => {
 
 export const WithWallet = () => (
   <Provider network={NETWORKS.rinkeby}>
-    <AddressUsingProvider />
+    <AddressUsingProvider shortened={false} />
+  </Provider>
+);
+
+export const WithWalletShortened = () => (
+  <Provider network={NETWORKS.rinkeby}>
+    <AddressUsingProvider shortened={true} />
```

Comment:
```suggestion
    <AddressUsingProvider shortened />
```

Omitting the value of the prop when it is explicitly `true` is recommended

Reply:
Ah I see, since it resolves to true it is not necessary to be explicit about it?